sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/SupportsNamespaces.java

```diff
@@ -41,34 +39,6 @@
 @Experimental
 public interface SupportsNamespaces extends CatalogPlugin {
 
-  /**
-   * A property to specify the location of the namespace. If the namespace
-   * needs to store files, it should be under this location.
-   */
-  String PROP_LOCATION = "location";
```

Comment:
These property names should remain in the public class, as we need the implementations to read and know them.

The only thing interval is which properties are reserved. We should move the `RESERVED_PROPERTIES` to `CatalogV2Utils`

Reply:
ah I see